docs/native-modules-android.md

```diff
@@ -532,6 +813,36 @@ params.putString("eventProperty", "someValue");
 sendEvent(reactContext, "EventReminder", params);
 ```
 
+</TabItem>
+<TabItem value="kotlin">
+
+```kotlin
+private fun sendEvent(reactContext: ReactContext, eventName: String, params: WritableMap?) {
+    reactContext
+      .getJSModule(DeviceEventManagerModule.RCTDeviceEventEmitter::class.java)
+      .emit(eventName, params)
+}
```

Comment:
One more thing. The Java example includes the imports at the beginning of code, do we need those in Kotlin file too?